tests/scripts/deploy-validate-vault.sh

```diff
@@ -164,8 +165,26 @@ function deploy_vault {
   sed -i "s|ROOK_TOKEN|$ROOK_TOKEN|" tests/manifests/test-kms-vault.yaml
 }
 
+function validate_rgw_token {
+  # Create secret for RGW server in kv engine
+  ENCRYPTION_KEY=$(openssl rand -base64 32)
+  kubectl exec vault-0 -- vault kv put -ca-cert /vault/userconfig/vault-server-tls/vault.crt rook/"$RGW_BUCKET_KEY" key="$ENCRYPTION_KEY"
+  RGW_POD=$(kubectl -n rook-ceph get pods -l app=rook-ceph-rgw | awk 'FNR == 2 {print $1}')
+  RGW_TOKEN_FILE=$(kubectl -n rook-ceph describe pods "$RGW_POD" | grep  "rgw-crypt-vault-token-file" | cut -f2- -d=)
+  VAULT_PATH_PREFIX=$(kubectl -n rook-ceph describe pods "$RGW_POD" | grep  "rgw-crypt-vault-prefix" | cut -f2- -d=)
+  VAULT_TOKEN=$(kubectl -n rook-ceph exec $RGW_POD -- cat $RGW_TOKEN_FILE)
+
+  #fetch key from vault server using token from RGW pod, P.S using -k for curl since custom ssl certs not yet to support in RGW
```

Comment:
What's the timeline to support signed certificates?

Reply:
PR posted in https://github.com/ceph/ceph/pull/37730, @mattbenjamin or @mdw-at-linuxbox can give an idea about time frame